src/pages/Storefront/Cart.js

```diff
@@ -55,21 +54,34 @@ const CartItems = ({ checkoutData }) => {
               />
               <Divider borderColor="brand.gray" pb={8} />
             </Box>
-          ))}
+          ))
+        ) : (
+          <VStack spacing={8}>
+            <Heading size="h3" color="black" py={0}>
```

Comment:
Random question but is there a difference between just writing out the text in all caps or using `textTransform="uppercase"`, I think I've seen both it used in the codebase

Reply:
I was wondering this too. I'm not sure what's better, I personally like using `textTransform`

Reply:
Going to change it to `textTransform` for consistency